trackers/core/ksptracker/tracker.py

```diff
@@ -0,0 +1,277 @@
+from dataclasses import dataclass
+from typing import Any, Dict, List, Union
+
+import networkx as nx
+import numpy as np
+import supervision as sv
+
+from trackers.core.base import BaseTracker
+
+
+@dataclass(frozen=True)
+class TrackNode:
```

Comment:
It seems like this implementation uses the detections themselves as graph nodes, while the paper proposes a graph where nodes represent a grid of discrete physical locations at every point in time which allows the tracker to interpolate trajectories through areas with missing detections.